BioseqSuite.py

```diff
@@ -0,0 +1,185 @@
+from collections import Counter
+from modules.calculate_protein import calculate_amino_acid_percentages, calculate_pI
+from modules.classify_amino_acid import classify_amino_acid
+from modules.counting_protein import counting_point_mutations, counting_molecular_weight, count_variant_rna
+from modules.find_amino_acid_indices import find_amino_acid_indices
+from modules.get_occurrences import get_occurrences
+from modules.is_protein import is_protein
+from modules.reverse_and_complement import reverse, complement, reverse_complement
+from modules.total_protein_charge import determine_total_protein_charge
+from modules.transcribe import transcribe
+import sys
+

```

Comment:
По PEP8, импорты надо располагать так (и не забыть после импортов две пустые строки)
Три группы импортов: встроенные библиотеки, внешние библиотеки (это типа pandas и пр., которые надо сначала в окружение устанавливать), собственные библиотеки. Все три группы разделяются пустой строкой. После импортов - две пустые строки
Да, и частичные импорты (from ___ import ___ ) после полных
```suggestion
import sys
from collections import Counter

from modules.calculate_protein import calculate_amino_acid_percentages, calculate_pI
from modules.classify_amino_acid import classify_amino_acid
from modules.counting_protein import counting_point_mutations, counting_molecular_weight, count_variant_rna
from modules.find_amino_acid_indices import find_amino_acid_indices
from modules.get_occurrences import get_occurrences
from modules.is_protein import is_protein
from modules.reverse_and_complement import reverse, complement, reverse_complement
from modules.total_protein_charge import determine_total_protein_charge
from modules.transcribe import transcribe




```